thoth/storages/graph/postgres.py

```diff
@@ -2261,6 +2263,9 @@ def get_si_aggregated_python_package_version(
                 index_url=index_url
             )
 
+            if query.count() == 0:
```

Comment:
I think this will introduce two queries - one for count and the other for actual data retrieval. It should be more optimal to move this check down bellow:

```python
result = query.order_by(SecurityIndicatorAggregatedRun.datetime.desc()).first()
if not result:
    return None

# ... result.to_dict() logic
```